backend/src/ledgers.ts

```diff
@@ -176,3 +263,13 @@ function formatDate(s: string): string {
   const day = `0${d.getUTCDate()}`;
   return `${month.slice(-2)}-${day.slice(-2)}`;
 }
+
+const getLedgerKey = {
+  hour: (closed_at: string) =>
```

Comment:
for the hours can we use just the minute as the key? this will make it easier for both the frontend to use and for debugging
eg. it would be:

```
[
  {
    "date": "57M",
    "transaction_count": 703,
    "operation_count": 1721
  },
  {
    "date": "56M",
    "transaction_count": 3833,
    "operation_count": 9317
  },
  {
    "date": "55M",
    "transaction_count": 3264,
    "operation_count": 8717
  },
```